proto/pulumi/language.proto

```diff
@@ -58,6 +58,23 @@ service LanguageRuntime {
     rpc Pack(PackRequest) returns (PackResponse) {}
 }
 
+// ProgramInfo are the common set of options that a language runtime needs to execute or query a program. This
+// is filled in by the engine based on where the `Pulumi.yaml` file was, the `runtime.options` property, and
+// the `main` property.
+message ProgramInfo {
+    // the root of the project, where the `Pulumi.yaml` file is located.
+    string root_directory = 1;
```

Comment:
Nit: Not that it really matters, but I see we use a mix of both camelCase and snake_case for field names in this file (and likely others). I don't love the inconsistency, but we're already in this mess. Is the plan to just use snake_case for all new fields going forward?

Reply:
Yeh snake case is the expected format for .proto files.